lava_test_plans/testcases/templates/android-xts.yaml.jinja2

```diff
@@ -26,5 +26,5 @@
         RESULTS_FORMAT: "{{xts_result_format}}"
         ANDROID_VERSION: "{{ANDROID_VERSION}}"
         TEST_REBOOT_EXPECTED: "{{xts_expect_reboot}}"
-      name: "{{xts_test_suite_name}}"
+      name: "{{test_name}}"
```

Comment:
just please make sure the tuxsuite side will set this test_name to the same value as passed in the tuxplan configuration file, otherwise it might be empty when the android-cts or android-vts is used.
And regarding to the naming, it might be better to use the upper case as it's value will be passed from out side. and I still think name it as "testsuite" name would be good, as it's shown in the lava as testsuite.